src/test/java/com/puppycrawl/tools/checkstyle/JavadocPropertiesGeneratorTest.java

```diff
@@ -355,4 +355,23 @@ public void testParseError() throws Exception {
         }
     }
 
+    @Test
+    public void testCorrect2(@SysErr Capturable systemErr, @SysOut Capturable systemOut)
```

Comment:
Make method name to describe a reason.
Same for Input file name.
And if more wording is required you can put it in comment of Input.

Reply:
done